boxsdk/network/logging_network.py

```diff
@@ -48,32 +55,216 @@ def _log_request(self, method, url, **kwargs):
         :type access_token:
             `unicode`
         """
-        self._logger.info(self.REQUEST_FORMAT, method, url, pformat(kwargs))
+        self._logger.info(self.REQUEST_FORMAT, {'method': method, 'url': url, 'request_kwargs': pformat(kwargs)})
 
-    def _log_response(self, response):
-        """
-        Logs information about the Box API response.
+    def _log_exception(self, method, url, exc_info):
+        """Log information at WARNING level about the exception that was raised when trying to make the request.
 
-        :param response: The Box API response.
+        :param method:  The HTTP verb that was used to make the request.
+        :type method:   `unicode`
+        :param url:   The URL for the request.
+        :type url:  `unicode`
+        :param exc_info:  The exception info returned from `sys.exc_info()`.
         """
-        if response.ok:
-            if response.request_response.raw is not None:
```

Comment:
`response.request_response.raw` is always set (both for `stream=True` and `stream=False`, so this code path was always being followed. cc @Jeff-Meadows